app/Resources/install/dataimport/SOM_competency_data.sql

```diff
@@ -0,0 +1,34 @@
+LOCK TABLES `competency` WRITE;
```

Comment:
I'm guessing all fo these .sql files were committed accidentally?  Otherwise I don't know what they are for?